packages/api/test/e2e/tx.e2e.ts

```diff
@@ -144,21 +185,25 @@ describe('e2e transactions', () => {
               events.forEach(({phase, event: {data, method, section}}) => {
                 console.log('\t', phase.toString(), `: ${section}.${method}`, data.toString());
               });
-
               done();
             }
           });
       });
 
       it('use signer', async done => {
-        const tx = api.tx.genericAsset.transfer(16000, receiver.address, 10000);
-        const txOpt = {
-          feeExchange: {
-            assetId: '16000',
-            maxPayment: '50000000000000000',
+        const feeExchange = {
+          assetId: feeAssetId,
+          maxPayment: '50000000000000000',
+        };
+        const transactionPayment = {
+          tip: 2,
+          feeExchange:  {
+            FeeExchangeV1: feeExchange,
           },
         };
-        return tx.signAndSend(sender.address, txOpt, ({events, status}) => {
+
+        const tx = api.tx.genericAsset.transfer(16001, receiver.address, 100);
+        return tx.signAndSend(sender.address, {transactionPayment}, ({events, status}) => {
```

Comment:
Does it work with `sender.Keypair` aswell?

Reply:
yes